### PR TITLE
Add publishing date article page

### LINKED ISSUE #285: Add the publishing date for the article page
Right now, the date is not shown on the article page. Add the publishing year.

### PR DESCRIPTION
**Pull request by jazellemaira**
This PR adds the publishing year for the article page. 

Fixes #285 

image

### CODE REVIEW

File: app/pages/articles/[articleId].tsx [hunk 1]
```diff
@@ -107,7 +108,7 @@ const ArticleDetails = (props) => {
         </div>
         <div className="article__author text-base text-center m-2 max-w-2xl text-gray-dark dark:text-gray-light">
           <FaCrown className="inline m-2" />
-          {article.authorString}
+          {article.authorString} ({article.publishedYear})
```
Comment (nsunami) [inline app/pages/articles/[articleId].tsx:111]: ` {article.authorString} {publishedYear}` will work and print out the string with the parenthesis already? Also, in the case there is no published year the filed will be blank, as opposed to showing as "()"

**Review by nsunami — CHANGES_REQUESTED** [review]
Do you think this change works?

File: app/pages/articles/[articleId].tsx (hunk 1 above)
Comment (jazellemaira) [reply app/pages/articles/[articleId].tsx:111]: ooh! that would be great! can you change it!

**Review by nsunami — APPROVED** [review]
LGTM ILY!

**Pull request merged**